jre/guide/java/java_jre_askgrantdialog_notinca/oval/jre.xml

```diff
@@ -0,0 +1,34 @@
+<def-group>
+  <definition class="compliance" id="java_jre_askgrantdialog_notinca" version="1">
+    <metadata>
+      <title>Disable Java Execution From Untrusted Sources</title>
+      <affected family="unix">
+        <product>Java Runtime Environment</product>
+      </affected>
+      <description>The dialog to enable users to grant permissions to execute
+      signed content from an un-trusted authority must be disabled.</description>
+    </metadata>
+    <criteria>
+      <criterion comment="Check deployment.security.askgrantdialog.notinca" test_ref="test_java_jre_askgrantdialog_notinca" />
+      <criterion comment="Check deployment.security.askgrantdialog.notinca.locked" test_ref="test_java_jre_askgrantdialog_notinca_locked" />
+    </criteria>
+  </definition>
+
+  <ind:textfilecontent54_test check="all" comment="Check deployment.security.askgrantdialog.notinca" id="test_java_jre_askgrantdialog_notinca" version="1">
+    <ind:object object_ref="object_java_jre_askgrantdialog_notinca" />
+  </ind:textfilecontent54_test>
+  <ind:textfilecontent54_object id="object_java_jre_askgrantdialog_notinca" version="1">
+    <ind:filepath>/etc/.java/deployment/deployment.properties</ind:filepath>
+    <ind:pattern operation="pattern match">^deployment.security.askgrantdialog.notinca=false$</ind:pattern>
+    <ind:instance datatype="int">1</ind:instance>
+  </ind:textfilecontent54_object>
+
+  <ind:textfilecontent54_test check="all" comment="Check deployment.security.askgrantdialog.notinca.lock" id="test_java_jre_askgrantdialog_notinca_locked" version="1">
+    <ind:object object_ref="object_java_jre_askgrantdialog_notinca_locked" />
+  </ind:textfilecontent54_test>
+  <ind:textfilecontent54_object id="object_java_jre_askgrantdialog_notinca_locked" version="1">
+    <ind:filepath>/etc/.java/deployment/deployment.properties</ind:filepath>
+    <ind:pattern operation="pattern match">^deployment.security.askgrantdialog.notinca.locked$</ind:pattern>
```

Comment:
As this is a separate configuration item, it should really be in its own rule. So, either we deal with this now, or next month when we plan to update the content for multiple products/versions as well as will be submitting to DISA.

Reply:
Can you provide an example of this? It is my understanding that each requirement should map to a single check and remediation. How do we reference multiple rules for a single requirement in the overlay? How would these then relate back to the STIG? If we were to import the results into the STIG checklists, the checks would no longer align with the requirements.

Reply:
@thenefield The ideal scenario would be to request a fix in the STIG itself from DISA. At this moment what we can do is just to split the configuration check into two and wait for an update to be able to assign a new STIG id.